city_scrapers/spiders/il_pollution_control.py

```diff
@@ -0,0 +1,203 @@
+import json
+import re
+from datetime import datetime
+from io import BytesIO
+from urllib.parse import urljoin
+
+import scrapy
+from city_scrapers_core.constants import BOARD, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from PyPDF2 import PdfFileReader
+
+
+class IlPollutionControlSpider(CityScrapersSpider):
+    name = "il_pollution_control"
+    agency = "Illinois Pollution Control Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["pcb.illinois.gov"]
```

Comment:
Small thing, but we'll be getting rid of `allowed_domains` going forward because we don't need to restrict our crawlers as much since they're pretty targeted. If you could drop this while fixing the other `urljoin` method that'd be great!

Reply:
Done!